app/views/content/assessment-only-providers.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Get qualified teacher status through an 'assessment only' programme
+title: QTS assessment only providers
```

Comment:
Just wondering whether this quite makes sense @EllieNodder ?

I know we are going to look at whether to consolidate the two AO pages but I'm not sure this h1/title is quite right in the meantime on reflection

Reply:
My thinking was that the /train-to-be-a-teacher/if-you-have-experience page is talking about how and why you can do assessment only, whereas this page is basically just listing the different providers, with a bit of preamble at the top. They go to the first page to find out what, then this page to find out where.

The url is also just /assessment-only-providers, so this would make the page title/h1 more in line with that.

What were you thinking?

Reply:
Sorry - my comment wasn't very helpful!

I was really meaning should it be 'Assessment only QTS providers' rather than QTS at the front

(NB most visitors land on the page rather than come from the other one first, looking at GA. Something to consider when we look at consolidation)

Reply:
ohhh, I see!! yes I think you're right - shall change